confd/service/service.go

```diff
@@ -0,0 +1,188 @@
+package service
+
+import (
+	"encoding/json"
+	"html/template"
+	"log"
+	"os"
+	"path"
+
+	. "github.com/cloudogu/ces-confd/confd"
+	"github.com/coreos/etcd/client"
+	"github.com/pkg/errors"
+	"golang.org/x/net/context"
+)
+
+// Services is a collection of service structs
+type Services []*Service
+
+// Service is a running service
+type Service struct {
+	Name string
+	URL  string
+}
+
+type Source struct {
```

Comment:
**[checkstyle]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Source should have comment or be unexported